pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -7573,6 +7575,23 @@ func TestChangefeedBackfillCheckpoint(t *testing.T) {
 		// Resume job.
 		require.NoError(t, jobFeed.Resume())
 
+		// Verify that the resumed job has restored the progress from the checkpoint
+		// to the change frontier.
+		timestampSpansMap := checkpoint2.ConvertLegacyCheckpoint(jobCheckpoint, hlc.Timestamp{}, hlc.Timestamp{})
+		testFrontier, err := resolvedspan.NewCoordinatorFrontier(jobCheckpoint.Timestamp, *h, tableSpan)
```

Comment:
nit: rename to `expectedFrontier` or something that communicates that this is the expected state